### PR TITLE
add codegen comment to top of files

### PR DESCRIPTION
**Pull request by kuhe**
Simpler version of https://github.com/awslabs/smithy-typescript/pull/535, adds a comment to files indicating generated code to discourage direct modification.

For detailed debugging, use the feature from https://github.com/awslabs/smithy/pull/1198

SDK changes: https://github.com/aws/aws-sdk-js-v3/pull/3592

### CODE REVIEW

**Comment by kuhe** [comment]
do I need to simultaneously merge the generate-clients diff into v3?

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# used with another locally situated consuming codebase to test changes.
+local:
+	./gradlew clean build publishToMavenLocal
+
+build:
+	./gradlew build
+
+# run tests
+test:
+	./gradlew test
```
Comment (AllanZhengYP) [inline Makefile:12]: What is this used for?

**Review by AllanZhengYP — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (kuhe) [reply Makefile:12]: I'm using it for shortcuts to commonly run commands.

**Review by AllanZhengYP — APPROVED** [review]
Ship again!

**Pull request merged**